types/for-each/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for for-each 0.3
+// Project: https://github.com/Raynos/for-each
+// Definitions by: Jordan Harband <https://github.com/ljharb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function forEach<T = unknown, THIS = undefined>(arr: T[], callback: (this: THIS, value: T, index: number, array: T[]) => void, thisArg?: THIS): void;
```

Comment:
```suggestion
declare function forEach<T = unknown, This = undefined>(arr: T[], callback: (this: This, value: T, index: number, array: T[]) => void, thisArg?: This): void;
```
Type parameters generally aren't in all caps.

Reply:
sure, i can change that - "T" is in all caps, so i figured that was the convention :-)